package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "antd": "^4.9.4",
+    "js-logger": "^1.6.1",
```

Comment:
i understand the usefulness of extra libraries like js-logger, lodash etc.. however, for this kind of pet project, i recommend using simple tools rather than whipping out the big guns. one of the skills senior developers value is using minimum dependencies (except when rushed, or when forced to use a convention with teams)

Reply:
It's good to be on budget so I'll remove `js-logger` 😆  I'll consider about `lodash` later. 

> one of the skills senior developers value is using minimum dependencies
Sound interesting! However, how do you know when to include a package or not? Or will it depend on the case? 🤔